static/app/views/settings/projectPlugins/projectPlugins.tsx

```diff
@@ -21,7 +21,7 @@ type Props = {
   error: React.ComponentProps<typeof RouteError>['error'];
   loading: boolean;
   onChange: React.ComponentProps<typeof ProjectPluginRow>['onChange'];
-  plugins: Plugin[];
+  plugins: Plugin[] | null;
```

Comment:
IMO since you converted to TypeScript you don't need that `plugins=null` test anymore, since TypeScript would ensure it never happens. Then we wouldn't need the `| null` here!

Reply:
yes you are right. I thought that maybe the types in the component were wrong due to the test and checks we have...but will revert it ...thanks